### ISSUE TITLE
Documentation on use outside of Apache mod_fcgi.

### ISSUE THREAD

**Issue by amcgregor** [Kozea/flipflop#3]
Some of us like to use Nginx, and right now the code itself is completely non-obvious as to how to get things running.

**Comment by samveen** [comment]
I looked into the code, and I found no configuration for opening a listening socket. From this, I gather that this is built purely for use with apache. There is no possible way to use this with Nginx.

**Comment by amcgregor** [comment]
Yup; I abandoned this fork in favour of https://github.com/chxanders/flup3 or another straight port of Flup to Python 3.

**Comment by golightlyb** [comment]
It works fine on Nginx. This is what I'm using:

In /etc/hosts

    127.0.0.1	yourwebsite.localhost

In /etc/nginx/sites-available/yourwebsite.localhost

    location ~ \.py$ {
        expires off;
        include fastcgi_params;
        fastcgi_pass 127.0.0.1:9002;
    }

To start:

    cwd=`pwd`
    spawn-fcgi -d "${cwd}" -a 127.0.0.1 -p 9002 "${cwd}/index.py"
    sudo service nginx start

To stop:

    sudo service nginx stop
    cwd=`pwd`
    kill `pgrep -f "python3 ${cwd}/index.py"`

**Comment by amcgregor** [comment]
That it requires an external, third-party tool to construct magic environment variables to make it work is unfortunate.  Please, please, please expose explicit configuration within the Python API, as Flup previously did.

**Comment by samveen** [comment]
@golightlyb You mean `It works fine on _INSERT_FAVOURITE_FCGI_CONTAINER_HERE_.`, not that it works fine `on` nginx. It needs a container.
